vendor/bat-native-ads/src/bat/ads/internal/account/deposits/deposit_info.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_DEPOSITS_DEPOSIT_INFO_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ACCOUNT_DEPOSITS_DEPOSIT_INFO_H_
+
+#include <string>
+
+#include "base/time/time.h"
+
+namespace ads {
+
+struct DepositInfo final {
+  DepositInfo();
+  DepositInfo(const DepositInfo& info);
```

Comment:
if copy constructor is added then copy operator should be also defined

Reply:
Removed as not required and raised https://github.com/brave/brave-browser/issues/22245 to fix elsewhere